lib/bap_image/objectFile.mli

```diff
@@ -0,0 +1,50 @@
+(** LLVM object file *)
+
+open Core_kernel.Std
+
+type t
+
+external create : Bigstring.t -> t Or_error.t =
+  "object_file_create_stub"
+
+type archType =
```

Comment:
ArchType implemented in Triple.h in llvm. It used with method ObjectFile::getArch. Idea is to create bap arch from llvm arch in ocaml code in Image_llvm.of_data method.